build.sbt

```diff
@@ -69,18 +66,19 @@ def coreProject(includePlayVersion: String): Project = {
   val scalaVersions = includePlayVersion match {
     case Play_2_5 => Seq(Scala_2_11)
     case Play_2_6 => Seq(Scala_2_11, Scala_2_12)
-    case Play_2_7 => Seq(Scala_2_12, Scala_2_13)
+    case Play_2_7 => Seq(Scala_2_11, Scala_2_12, Scala_2_13)
   }
   val path = s"play$playSuffix-core"
   commonProject(path, path).settings(
     name := s"play$playSuffix-test-ops-core",
+    scalaVersion := scalaVersions.head,
     crossScalaVersions := scalaVersions,
     // fail the build if the coverage drops below the minimum
     coverageMinimum := 80,
     coverageFailOnMinimum := true,
     // add library dependencies
     resolvers ++= Seq(
-      "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/",
+      "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/",
```

Comment:
I've been deleting this from all our repositories in favor of proxying everything through Artifactory. This resolver has proven to be pretty flaky.